R/tar_resources_gcp.R

```diff
@@ -0,0 +1,39 @@
+#' @title Target resources: GCP storage formats
```

Comment:
Would you write the full name "Google Cloud Platform" somewhere in the title. Just occurred to me that I should also do this for AWS.